Library/Formula/libuuid.rb

```diff
@@ -5,6 +5,7 @@ class Libuuid < Formula
   url "https://downloads.sourceforge.net/project/libuuid/libuuid-1.0.3.tar.gz"
   sha256 "46af3275291091009ad7f1b899de3d0cea0252737550e7919d17237997db5644"
 
+  conflicts_with "libuuid", :because => "both install lib/libuuid.a"
```

Comment:
Hey @sjackman - please pardon my commenting on an already-merged pull request, but was this conflicts_with supposed to be in the util-linux formula? The way I see it, the libuuid formula is currently conflicting with itself.